### PR TITLE
Dsr 48 pubsub repository maintenance

### PR DESCRIPTION
**Pull request by TWeatherston**
- Merged in unit tests and poetry from other branches
- Added github actions to lint, test and publish
- Added a readme file
- Relaxed dependencies for Celery and Kombu

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[run]
+omit = */__init__.py, *tests/*, *commands.py, *commands/*
```
Comment (sluenenglish) [inline .coveragerc:2]: Should we exclude commands?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: 2
+
+updates:
```
Comment (sluenenglish) [inline .github/dependabot.yml:3]: Personally, I don't think the PRs are useful, and just the monitoring is useful. What do you think?

File: .github/workflows/tests.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```
Comment (sluenenglish) [inline .github/workflows/tests.yaml:20]: Might be worth using something like tox to test multiple versions?

File: .github/workflows/publish.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@master
+        with:
+          python-version: 3.7
```
Comment (sluenenglish) [inline .github/workflows/publish.yml:17]: Maybe use latest-1 python version = 3.9 ?

File: .coveragerc (hunk 1 above)
Comment (TWeatherston) [reply .coveragerc:2]: We do already test all the functionality inside the commands so we would essentially just be testing that calling one function calls another. Happy to add that in though if you think it's worthwhile having?

File: .github/dependabot.yml (hunk 2 above)
Comment (TWeatherston) [reply .github/dependabot.yml:3]: I agree that the PRs aren't particularly helpful. So long as we're being notified somewhere useful about any critical security updates!?

File: .github/workflows/tests.yaml (hunk 3 above)
Comment (TWeatherston) [reply .github/workflows/tests.yaml:20]: Yeah that would be nice, I'll add that :+1:

File: .github/dependabot.yml (hunk 2 above)
Comment (sluenenglish) [reply .github/dependabot.yml:3]: we are :)

**Review by sluenenglish — APPROVED** [review]

**Pull request merged**